### DISCUSSION TITLE
Tokenizer somehow splits the word "id" into 2 sepperate tokens

### DISCUSSION THREAD

**Discussion by CraiyDev** [explosion/spaCy#10570] [Help: Other Questions] [answered]
If you simply call `nlp("search for the id 10")` you will notice that the word "id" gets split into 2 sepperate tokens named i and d
This behavior happens for _en_core_web_md_ as well as _en_core_web_sm_, I havent tried more but it seems to be a general problem with tokenization

**Comment by adrianeboyd** [comment, marked as answer]
By default the English tokenizer settings treat "id" as a variant of "I'd". Here's a comment about how to modify this: https://github.com/explosion/spaCy/issues/10455#issuecomment-1061512805
